examples/CMakeLists.txt

```diff
@@ -40,6 +32,7 @@ function(cub_add_example target_name_var example_name example_src cub_target)
   add_executable(${example_target} "${example_src}")
   target_link_libraries(${example_target} ${cub_target})
   cub_clone_target_properties(${example_target} ${cub_target})
+  cub_configure_cuda_target(${example_target} RDC ${CUB_FORCE_RDC})
```

Comment:
I personally don't like examples that can't be built with an installed version of a project. I would recommend dropping usage of `cub_clone_target_properties` and `cub_configure_cuda_target`

Reply:
This is a good consideration, but outside of the scope of this PR and is an existing issue. We can open an issue.